libs/service-portal/signature-collection/src/lib/constants.ts

```diff
@@ -1,6 +1,6 @@
 export const CollectionType = {
   Presidential: 'Forsetakosningar',
-  Parliamentary: 'Alþingiskosningar',
+  Parliamentary: 'Alþingiskosning NÝTT TEST',
```

Comment:
Why this value? Just curious

Reply:
Því þetta er valuið sem er að koma frá Þjóðskrá - það er verið að gera string comparison á "name" 🫠

Reply:
refactoraði - sæki og nota núna isPresidential flaggið á currentCollection